### DISCUSSION TITLE
Visually distinguish Battlesnakes close to the board background color

### DISCUSSION THREAD

**Discussion by fables-tales** [BattlesnakeOfficial/feedback#108] [Bug Reports]
E.g. it's almost impossible to visually distinguish whitish-meteor here, and not that I think they should be forced to change their color, but it'd be really good for them to be more visible on the board like this 
![image](https://user-images.githubusercontent.com/96662/134783688-44b02983-6712-4497-8ece-bfeb4192552d.png)

**Comment by bvanvugt** [comment]
Yeah! It's a known issue and thanks for recording it here. Darcy's been playing with some ideas, but we haven't landed on one we love yet... here's the current leading proposal:
<img width="592" alt="Screen Shot 2021-09-26 at 1 59 36 PM" src="https://user-images.githubusercontent.com/1531419/134824023-97b3e4bd-11dd-4665-ae44-83b136c7d60f.png">
